akka-actor-tests/src/test/java/akka/pattern/PatternsTest.java

```diff
@@ -454,9 +454,10 @@ public void testCSAfterDuration() throws Exception {
     @Test
     public void testGracefulStop() throws Exception {
         ActorRef target = system.actorOf(Props.create(StopActor.class));
-        Future<Boolean> result = Patterns.gracefulStop(target, FiniteDuration.apply(200, TimeUnit.MILLISECONDS));
+        Future<Boolean> result = Patterns.gracefulStop(target,
+                scala.concurrent.duration.FiniteDuration.apply(200, TimeUnit.MILLISECONDS));
 
-        Boolean actual = Await.result(result, FiniteDuration.apply(3, SECONDS));
+        Boolean actual = Await.result(result, scala.concurrent.duration.FiniteDuration.apply(3, SECONDS));
```

Comment:
`FiniteDuration` doesn't cause a name conflict with Java `Duration` and this isn't sample code, just regular tests, so you can keep it imported here.